### PR TITLE
A5 resize benchmark

### PR DESCRIPTION
**Pull request by mcres**
This PR corresponds to issue #19

### CODE REVIEW

**Comment by mcres** [comment]
Regarding the last commit: is there any reason for the `trace*` files to be `.gitignore`d? 

https://github.com/robotperf/benchmarks/blob/bb219e1cf1c191a1c907efbb96abdb58d2cba71c/.gitignore#LL2C1-L2C7

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .DS_Store
 trace*
```
Comment (vmayoral) [inline .gitignore:2]:
```suggestion
```

File: benchmarks/perception/a5_resize/benchmark.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  
+  # Build the benchmark
+  colcon build --merge-install --packages-up-to a5_resize
+  
+  # Source the workspace as an overlay, launch the benchmark
+  source install/setup.bash
+  RMW_IMPLEMENTATION=rmw_cyclonedds_cpp ros2 launch a5_resize trace_a5_resize.launch.py
+
+# Data not valid at the moment, just a placeholder for now
+results:
```
Comment (vmayoral) [inline benchmarks/perception/a5_resize/benchmark.yaml:30]: Modify this and include results you obtained, for the hardware you used. See https://github.com/robotperf/benchmarks/blob/main/benchmarks/perception/a1_perception_2nodes/benchmark.yaml for inspiration.

File: benchmarks/perception/a5_resize/package.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>a5_resize</name>
+  <version>0.1.0</version>
+  <description>
+    A simple perception resize ROS robotics operation. 
+    Used to demonstrate a simple perception component 
+    image_pipeline package.
+  </description>
+  <maintainer email="martinho@todo.todo">martinho</maintainer>
```
Comment (vmayoral) [inline benchmarks/perception/a5_resize/package.xml:11]: Add your mail properly.

File: benchmarks/perception/a5_resize/benchmark.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+id: a5
+name: a5_resize
+description: |
+  A simple perception resize ROS robotics operation. Used to demonstrate a simple perception component [image_pipeline](https://github.com/ros-perception/image_pipeline) package.
+short: Perception resize ROS Component.
+graph: ../../../imgs/a5_resize.svg
```
Comment (vmayoral) [inline benchmarks/perception/a5_resize/benchmark.yaml:6]: This file doesn't exist in this PR. I can see `a2_rectify.svg`. Can you produce the graph and modify it accordingly?

**Comment by vmayoral** [comment]
@mcres other than the issue with the image above, all good.

**Comment by mcres** [comment]
Both a2 and a5 README have their own computational graphs now

**Comment by vmayoral** [comment]
Thanks @mcres!

**Pull request merged**